index.js

```diff
@@ -99,31 +99,25 @@ const build = (tagName, attrs, children) => {
 	if (setHTML && setHTML.__html) {
 		el.innerHTML = setHTML.__html;
 	} else {
-		children.forEach(child => {
-			el.appendChild(child);
-		});
+		el.appendChild(children);
 	}
 
 	return el;
 };
 
 function h(tagName, attrs) {
-	attrs = attrs || {};
+	const childrenArgs = [].slice.apply(arguments, [2]);
```

Comment:
Perhaps `[].slice.call(arguments, 2)` would be better?

Reply:
https://github.com/petkaantonov/bluebird/wiki/Optimization-killers#what-is-safe-arguments-usage

Reply:
Ok.